src/Uno/Prism.Uno/Prism.Uno.csproj

```diff
@@ -0,0 +1,85 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;xamarinios10;monoandroid90;xamarinmac20;uap10.0.16299</TargetFrameworks>
```

Comment:
Not yet, this is coming for Uno 2.3 or 2.4, either way, the dependency at this point won't change much, it can be kept as 9.0 until it fails to build in Prism itself.